### ISSUE TITLE
List items within p-autocomplete control are not read by screen reader. 

### ISSUE THREAD

**Issue by ranjanmicrosoft** [primefaces/primeng#8275]
Screen reader should provide proper information about the list items controls when using p-autocomplete. 

Issue exists in Chrome / Edge with Jaws / NVDA / Narrator

**Comment by cagataycivici** [comment]
Unable to replicate with PrimeNG 9, if the issue persists please create a new ticket with a test case reproducing the issue e.g. stackblitz or a github repo and it will be reviewed by our team once again.
